config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-release-0-4.yaml

```diff
@@ -0,0 +1,315 @@
+presubmits:
+  kubernetes-sigs/cluster-api-provider-azure:
+  - name: pull-cluster-api-provider-azure-test-release-0-4
+    always_run: true
+    optional: false
+    decorate: true
+    path_alias: "sigs.k8s.io/cluster-api-provider-azure"
+    branches:
+      # The script this job runs is not in all branches.
+      - ^release-0.4$
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20210201-77bfce8-1.18
+        command:
+        - "./scripts/ci-test.sh"
+    annotations:
+      testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
+      testgrid-tab-name: pr-test-release-0-4
+      testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
+  - name: pull-cluster-api-provider-azure-build-release-0-4
+    always_run: true
+    optional: false
+    decorate: true
+    path_alias: "sigs.k8s.io/cluster-api-provider-azure"
+    branches:
+      # The script this job runs is not in all branches.
+      - ^release-0.4$
```

Comment:
do we also need to skip the 0.4 branch for all the "main" jobs?

Reply:
it already has `base_ref: release-0-4` and the capi one doesn't have this label

Reply:
oh you main in the main jobs, I misunderstood

Reply:
fixed it